src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php

```diff
@@ -26,4 +31,19 @@ public function pongAction()
 
         return new Response($json, 200, ['Content-Type' => 'application/json']);
```

Comment:
JsonResponse everywhere

Reply:
I am going to leave that the way it is for now as I can't seem to get JMS to normalise the entity to an array for the response. We can revisit this at another time :-D